src/reconcile/rec_write.c

```diff
@@ -2182,6 +2185,37 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
     mod = page->modify;
     ref = r->ref;
 
+    /*
+     * If using the lookaside table eviction path and we found updates that weren't globally visible
+     * when reconciling this page, copy them into the database's lookaside store.
+     */
+    if (F_ISSET(r, WT_REC_LOOKASIDE))
+        WT_RET(__rec_las_wrapup(session, r));
+
+    /*
+     * Wrap up overflow tracking. If we are about to create a checkpoint, the system must be
+     * entirely consistent at that point (the underlying block manager is presumably going to do
+     * some action to resolve the list of allocated/free/whatever blocks that are associated with
+     * the checkpoint).
+     */
+    WT_RET(__wt_ovfl_track_wrapup(session, page));
+
+    /*
+     * If using the history store table eviction path and we found updates that weren't globally
+     * visible when reconciling this page, copy them into the database's history store. This can
+     * fail, so try before clearing the page's previous reconciliation state.
+     */
+    if (F_ISSET(r, WT_REC_HS))
+        WT_RET(__rec_hs_wrapup(session, r));
```

Comment:
The HS changes need to be removed. I think they got introduced by mistake.

Reply:
Yes, I think something went wrong with the cherry-pick, I will fix these.